src/App.js

```diff
@@ -1,19 +1,33 @@
-import {useState, useEffect} from 'react';
+import {useState, useEffect, useCallback, useRef} from 'react';
 import ReactMapGL from 'react-map-gl';
 import styled from 'styled-components';
 import 'mapbox-gl/dist/mapbox-gl.css';
 
 import Header from './components/Header/Header';
-import MapLayer from './components/MapLayer/MapLayer';
+import MarkerLayer from './components/MarkerLayer/MarkerLayer';
 import ParticipantCard from './components/Participant/ParticipantCard';
 
 const {REACT_APP_MAPBOX_TOKEN} = process.env;
 
 export default function App() {
+  const initialViewState = {
+    longitude: -83.0475,
+    latitude: 42.3316,
+    zoom: 8,
+    pitch: 0,
+    bearing: 0,
+  };
```

Comment:
Since this is something that won't change and will always be the same, it doesn't have to run every time the App component is called (every rerender). For this reason it is common to place such constants outside of the actual function call. Like this:
```suggestion
const {REACT_APP_MAPBOX_TOKEN} = process.env;

  const initialViewState = {
    longitude: -83.0475,
    latitude: 42.3316,
    zoom: 8,
    pitch: 0,
    bearing: 0,
  };

export default function App() {
```